src/Report/Xml/Source.php

```diff
@@ -0,0 +1,44 @@
+<?php declare(strict_types = 1);
+/*
+ * This file is part of the php-code-coverage package.
+ *
+ * (c) Sebastian Bergmann <sebastian@phpunit.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+namespace SebastianBergmann\CodeCoverage\Report\Xml;
```

Comment:
Well, php-cs-fixer isn't total. I was just referring to the fact that in the other files in this repo, there is always an extra newline there:

![image](https://cloud.githubusercontent.com/assets/2829600/24803660/cee35be0-1ba3-11e7-84c1-c6df58cfa9a4.png)